website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-upload-image-to-artifact-store
+page_title: Uploading Images to Artifact Stores
+description: |-
+  ...
+---
+
+# Uploading Images to Artifact Stores
```

Comment:
The title sounds like this is meant to be a more generic guide, but it seems to be TeamCity and S3-specific.